CMakeLists.txt

```diff
@@ -3,7 +3,9 @@
 #
 
 cmake_minimum_required(VERSION 2.6)
-
+if(CMAKE_VERSION VERSION_GREATER 2.8.12)
+  cmake_policy(SET CMP0048 OLD)
+endif()
```

Comment:
Instead of silencing the warning, can you instead fix the code so it doesn't emit the warning any more?